app.py

```diff
@@ -335,4 +272,26 @@ def get_items():
         return jsonify({'message': 'No items found'}), 404
     return jsonify([item.serialize() for item in items]), 200
 
+@app.route('/animalInfo', methods=['GET'])
```

Comment:
Just add this whole endpoint to the `create_app` function and then your test file won't need to import the app and create a new pytest fixture